src/juicy-popover.html

```diff
@@ -128,7 +128,7 @@
                 }
             }.bind(this));
 
-            expandable.addEventListener("mouseup", function (ev) {
```

Comment:
Isn't it enough to remove this handler completely and move the line 121 after `if { ... }`?